apps/server/lib/http/routes.ts

```diff
@@ -348,7 +351,7 @@ export const attachRoutes = (
 			}
 
 			return response.status(200).json({
-				access_token: createSessionResult.data.id,
+				access_token: (createSessionResult.data as any).id,
```

Comment:
Another `as any` cast

Reply:
@Ereski `waitResults` should be generic and accept return type right?

Reply:
@karaxuna which `waitResults`?

Reply:
@Ereski https://github.com/product-os/jellyfish/blob/master/apps/server/lib/http/routes.ts#L341

Reply:
How can you be sure that it is impossible for `createSessionResult.data` to be of any type other than object? https://github.com/product-os/jellyfish-types/blob/master/lib/queue/contracts/execute.ts#L16-L24

Reply:
Not sure how it works internally. It may return value with one of these types, in which case we don't handle all of them properly and not only typing is a problem, otherwise it's just typing problem and we should tell `waitResults` what type we expect to be returned.

Reply:
@karaxuna sounds like this piece of code needs some extra error checking instead of a blanket `as any`